src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -244,7 +244,7 @@ private async Task<IHttpActionResult> DeleteFunctionSecretAsync(string keyName,
             {
                 {
                     // System keys cannot be deleted.	
-                    return BadRequest("Invalid key name."); return BadRequest("Invalid key name.");
```

Comment:
Formatting looks off - you have an extra set of braces? Also, remove the "System keys cannot be deleted." comment since it no longer applies here

Reply:
thanks. bad merge. fixed it.